tasks/config-gitlab-runner.yml

```diff
@@ -1,37 +1,62 @@
 ---
-- name: Set advanced configuration | global section
+- name: Set advanced configuration | global section values
   lineinfile:
     dest: /etc/gitlab-runner/config.toml
-    regexp: '^(.*){{ item.param }} ='
-    line: '\1{{ item.param }} = {{ item.value }}'
+    regexp: '^(.*){{ item.key }} ='
+    line: '\1{{ item.key }} = {{ item.value }}'
     state: present
     backrefs: true
   notify: restart gitlab-runner
-  loop:
-    - param: concurrent
-      value: '{{ global_section.concurrent }}'
-  vars:
-    global_section: '{{ gitlab_runner_config.global }}'
+  with_dict: "{{ gitlab_runner_config.global_values | default({}) }}"
   become: true
+  tags:
+    - always
+
+- name: Set advanced configuration | global section strings
+  lineinfile:
+    dest: /etc/gitlab-runner/config.toml
+    line: '{{ item.key }} = "{{ item.value }}"'
+    state: present
+  notify: restart gitlab-runner
+  with_dict: "{{ gitlab_runner_config.global_strings | default({}) }}"
+  become: true
+  tags:
+    - always
 
 - name: Force all notified handlers to run
   meta: flush_handlers
+  tags:
+    - always
 
 - name: List all configured runners
   command: gitlab-runner list
   register: configured_runners
   failed_when: configured_runners.rc != 0
   changed_when: false
+  tags:
+    - always
+
+- name: disabling unsecure logging by default (override using tag 'unsecure_log')
+  set_fact:
+    no_logs: true
+  tags:
+    - always
+
+- name: enabling unsecure logging
+  set_fact:
+    no_logs: false
```

Comment:
I think we could create a new variable for manage logging and don't use tags

Reply:
> I think we could create a new variable for manage logging and don't use tags

I think tags are a better choice instead of variables because the main purpose is act differently specifying a single value in the cmd and I think are even easier to remember.

Reply:
Is using the vars be more elegant way to get same result and will take less extra code?
```
ansible-playbook playbook.yml --extra-vars "no_logs=true"  --tags ...
```

Reply:
Anyway this approach should be documented in readme.

Reply:
Both approaches are fine for me, I used the tag because I see the variable approach to be more suitable for overriding the values at inventory level, instead to change the behaviour of the playbook, the tag should be more correct. The amount of code added I don't think cause any problem.
I can add it to the documentation once it's decided the approach.

Reply:
@pavelpikta @vutkin  how should we implement it?
Let's decide and I'll make the changes.

Reply:
I think, using vars is more flexible, since you can define them in various ways: inventory, playbook, extra-vars etc.

Reply:
done, updated also the documentation adding the variable